styles/ethos-2-theme.css

```diff
@@ -970,7 +970,7 @@
   --qui-alias-color-background-selected-default: var(--qui-color-union-blue-200, #989fbb);
   --qui-alias-color-background-selected-hover: var(--qui-color-union-blue-100, #e6e8ee);
   --qui-alias-color-background-selected-press: var(--qui-color-absolutes-white, #ffffff);
-  --qui-alias-color-border-default: var(--qui-color-union-blue-200, #989fbb);
+  --qui-alias-color-border-default: var(--qui-color-union-blue-400, #2f3c75);
```

Comment:
we should not change their `ethos-2-theme.css`, if a block needs a different border colour, this should be handled at block level and not changing the global styles.

I'm thinking of something like:
```
.union-palette .cards > ul > li,
.cards.union-palette  > ul > li {
... set border colour
}
```

Reply:
fixed.